src/cockpit/packages.py

```diff
@@ -159,12 +183,18 @@ def filename_variants(filename, locale):
 
     def negotiate_file(self, path, headers):
         dirname, sep, filename = path.rpartition('/')
-        locale = headers.get('Accept-Language', '').split(',')[0].strip()  # obviously fix this
-
-        for variant in self.filename_variants(filename, locale):
-            logger.debug('consider variant %s', variant)
-            if Path(f'{dirname}{sep}{variant}') in self.files:
-                return f'{dirname}{sep}{variant}'
+        locales = []
+        accept_language = headers.get('Accept-Language', '')
+        if accept_language:
```

Comment:
...and change this to `accept_language is not None`.  No?  Or maybe even one better: 

```python
if isinstance(accept_language, str):
```

Reply:
Brrr, I rather have None then.